assets/js/pages/ClusterDetails/SBDDetails.test.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { sbdDevicesFactory } from '@lib/test-utils/factories';
+
+import { capitalize } from 'lodash';
+
+import SBDDetails from './SBDDetails';
+
+describe('SBDDetails', () => {
+  it('should render empty state message when sbd devices are empty', () => {
+    const expectedEmptyStateMsg = 'No additional fencing details to display.';
+    const sbdDevices = [];
+
+    render(<SBDDetails sbdDevices={sbdDevices} />);
+    expect(screen.getByText(expectedEmptyStateMsg)).toBeInTheDocument();
+  });
+
+  it('should render healthy sbd device', () => {
+    const expectedSBD = sbdDevicesFactory.buildList(1, { status: 'healthy' });
+    const [{ device: expectedDeviceName, status: expectedHealthStatus }] =
+      expectedSBD;
+    const capitalizedHealthStatus = capitalize(expectedHealthStatus);
+    render(<SBDDetails sbdDevices={expectedSBD} />);
+    expect(screen.getByText(expectedDeviceName)).toHaveTextContent(
```

Comment:
This double kind of expectation is really strange.
Here you should do `expect(screen.getByText(expectedDeviceName)).toBetTruthy()` or similar. Check that the text you got with a string contains that strings doesn't make much sense.
The same applies to other expectations